src/operations/create_collection.ts

```diff
@@ -73,7 +84,9 @@ export interface CreateCollectionOptions extends CommandOperationOptions {
   pkFactory?: PkFactory;
   /** A document specifying configuration options for timeseries collections. */
   timeseries?: TimeSeriesCollectionOptions;
-  /** The number of seconds after which a document in a timeseries collection expires. */
+  /** A document specifying configuration options for clustered collections. */
+  clusteredIndex?: ClusteredCollectionOptions;
```

Comment:
@baileympearson See creating a clustered collection involves specifying the clustered index options at collection creation time. It isn't possible to create a clustered index normally. Similar to timeseries (as the feature seems to have been extracted from there) except for with timeseries the indexes are hidden and not returned with listIndexes.

For clustered indexes (in clustered collections) the index shows up in listIndexes.